### DISCUSSION TITLE
Themes (personalization)

### DISCUSSION THREAD

**Discussion by PARZI-Y** [tranxuanthang/lrcget#104] [General]
White burns my eyes at night, a dark mode or being able to vary the colors to taste would be great. the rest is perfect

**Comment by Zedniac** [comment]
I agree. Not an urgent need since (at least in my use) LRCGET is the kind of app you open briefly to run and then let it go in the background for a while, but the bright white background with pink highlights can be rather blinding for those of us used to looking at dark colors all the time :)

**Reply by tranxuanthang** [reply]
There’s a work-in-progress on the `wip-dark-mode` branch, but it’s still far from being finished.

![Screenshot_20240924_223048](https://github.com/user-attachments/assets/52ed1583-6d7c-41bc-8911-423a2913770f)

**Comment by corylowry12** [comment]
#137
